qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -362,6 +364,76 @@ def test_scrub_repair(self):
         # fragstat should be fixed
         self.mount_a.run_shell(["rmdir", test_dir])
 
+    def test_scrub_merge_dirfrags(self):
+        """
+        That a directory is merged during scrub.
+        """
+
+        test_path = "testdir"
+        abs_test_path = "/{0}".format(test_path)
```

Comment:
```suggestion
        abs_test_path = f"/{test_path}"
```

Reply:
Done. I did it that way, as other places in the test class were similar. That said, it's probably best to use f-strings consistently within a function.